neuralhydrology/modelzoo/attention.py

```diff
@@ -0,0 +1,180 @@
+import logging
```

Comment:
I guess this file is a backup that was accidentally added? Looks like some old version of transformer.py